README.md

```diff
@@ -28,68 +27,65 @@ This package implements basic functionality (creating an H2OContext, showing the
-|               | 1.6.8                   | 3.10.0.7    | "rel-turing"     |        "7"         |
+The **rsparkling** R package requires the **sparklyr** and **h2o** R packages to run, so below we show how to install each of these packages.
 
-**NOTE**: A call to `h2o_release_table()` will display the above table in your R console.
+### Install sparklyr
 
-To install any one of the above versions you can use the following code snippet:
+We recommend the latest stable version of [sparklyr](http://spark.rstudio.com/index.html).
 
 ```r
-#We will install the H2O release `rel-turnbull` version `2` (H2O Version: 3.10.1.2)
+install.packages("sparklyr")
+```
 
-#The following two commands remove any previously installed H2O packages for R.
-if ("package:h2o" %in% search()) { detach("package:h2o", unload=TRUE) }
-if ("h2o" %in% rownames(installed.packages())) { remove.packages("h2o") }
-  
-# Next, we download packages that H2O depends on.
-if (! ("methods" %in% rownames(installed.packages()))) { install.packages("methods") }
-if (! ("statmod" %in% rownames(installed.packages()))) { install.packages("statmod") }
-if (! ("stats" %in% rownames(installed.packages()))) { install.packages("stats") }
-if (! ("graphics" %in% rownames(installed.packages()))) { install.packages("graphics") }
-if (! ("RCurl" %in% rownames(installed.packages()))) { install.packages("RCurl") }
-if (! ("jsonlite" %in% rownames(installed.packages()))) { install.packages("jsonlite") }
-if (! ("tools" %in% rownames(installed.packages()))) { install.packages("tools") }
-if (! ("utils" %in% rownames(installed.packages()))) { install.packages("utils") }
-  
-# Now we download, install and initialize the H2O package for R.
-install.packages("h2o", type="source", repos=(c("http://h2o-release.s3.amazonaws.com/h2o/rel-turnbull/2/R"))
+#### Install Spark via sparklyr
+
+The **sparklyr** package makes it easy to install any particular version of Spark.  Prior to installing **h2o** and **rsparkling**, the user will need to decide which version of Spark they would like to work with, as the remaining installation revolve around a particular major version of Spark (1.6 vs 2.0).  
 
+The following command will install Spark 2.0.0:
+
+``` r
+library(sparklyr)
+spark_install(version = "2.0.0")
 ```
 
-The example below will use Spark version 2.0, Sparkling Water version 2.0.3, & H2O version 3.10.1.2.
 
-## Installation
+### Install h2o
 
-The **rsparkling** R package requires the **sparklyr** and **h2o** R packages to run.
+**rsparkling** currently requires that a certain version of H2O be used, depending on which major version of Spark is used, although this requirement will be relaxed in a future version.  Each release of Sparking Water is built from specific versions of H2O, and those versions are listed in the table below.
 
-### Install sparklyr
+Most users will want to use the most recent compatible Sparkling Water compatible version of H2O would be used.
```

Comment:
I would change this to: ``rsparkling` will pick up the latest Sparkling Water based on the major Spark version provided.`